### PR TITLE
Bug 1198264 - Check in regression test for bug 1190791

### PR DESCRIPTION
**Pull request by mwargers**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=366e0c90c1933788957f3ee2a83e30c223c09baf

File: tests/python/gaia-ui-tests/gaiatest/tests/functional/test_wifi_multiple_connect.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+
+class TestWiFiMultipleConnect(GaiaTestCase):
+
+    @parameterized("1st_run")
+    @parameterized("2nd_run")
+    # The second run is necessary to check for the regression bug this
+    # would cause, namely an unusable device, see bug 1190791.
+    def test_connect_and_forget_all_networks(self):
+        for x in range(0, 30):
```
Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/test_wifi_multiple_connect.py]: 30 is a magic number. I'd also comment on the reason why we have this value, and not another like 10

Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/test_wifi_multiple_connect.py]: It is needed for this regression to occur, I'll add it to the test.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=775ea2ffe2b886c8e8f94b474761fab45c8c97a3

**Pull request merged**